app/routes/wallet/credentials.py

```diff
@@ -136,38 +235,96 @@ async def get_credential_revocation_status(
     return result
 
 
-@router.get("/w3c", response_model=VCRecordList)
+@router.get(
+    "/list/w3c",
+    response_model=VCRecordList,
+    summary="Fetch a list of W3C credentials from the wallet",
+)
 async def list_w3c_credentials(
-    count: Optional[str] = None,
-    start: Optional[str] = None,
-    wql: Optional[str] = None,
-    body: Optional[W3CCredentialsListRequest] = None,
+    contexts: Optional[List[str]] = Query(None),
+    types: Optional[List[str]] = Query(None),
+    schema_ids: Optional[List[str]] = Query(None),
+    subject_ids: Optional[List[str]] = Query(None),
+    proof_types: Optional[List[str]] = Query(None),
+    issuer_id: Optional[str] = Query(None),
+    given_id: Optional[str] = Query(None),
+    tag_query: Optional[str] = Query(None),
+    max_results: Optional[int] = Query(None),
```

Comment:
Let's trim all these options down to: schema_ids, maybe issuer_id, and max_results

Question - does issuer_id == issuer_did? Let's check in ACA-Py or the responses if that's the case ... because we can name it `issuer_did` for consistency.

Similarly, I guess `max_results` is like `limit`. So let's call it limit for consistency. I know that ACA-Py needs a contribution here for the pagination feature too, but that can come later